iron-overlay-behavior.html

```diff
@@ -15,7 +15,7 @@
 <link rel="import" href="iron-overlay-manager.html">
 
 <script>
-
+'use strict';
```

Comment:
This is an old argument, and a bike shed trap. For example, please refer to [this issue](https://github.com/twbs/bootstrap/issues/3057) about semi-colons (or lack thereof) in Bootstrap sources. If you ask me, it is not the responsibility of tools to make guesses about how the code should be formatted, but this is my opinion. Please use your best judgement!